lib/util/aws/cloudfront_signed_url.ex

```diff
@@ -48,8 +77,47 @@ defmodule Antikythera.Aws.CloudfrontSignedUrl do
     ]
   end
 
-  defunp create_signature(policy_statement :: v[String.t], private_key :: v[String.t]) :: String.t do
-    :public_key.sign(policy_statement, :sha, decode_rsa_key(private_key))
+  defunpt generate_ip_address_string(addresses :: v[[String.t]]) :: v[String.t] do
+    case addresses do
+      [address]     -> ~s/"#{String.trim(address)}"/
+      [address | t] -> Enum.reduce(t, ~s/["#{String.trim(address)}"/, fn (address, result) -> result <> ~s/,"#{String.trim(address)}"/ end) <> "]"
```

Comment:
(optional)
When this function is called, `addresses` aren't empty, so you could write:
```elixir
case addresses do
  [address] -> ~s/"#{String.trim(address)}"/
  addresses -> "[" <> Enum.map_join(addresses, ",", fn address -> ~s/"#{String.trim(address)}"/ end) <> "]"
end
```

Reply:
It's better than the current code. Fixed at https://github.com/access-company/antikythera/pull/131/commits/0fee513181c93d0d4971c6df9736e4ec3ee2c0dd